news/2023/2023-10-22-halloween-descends.md

```diff
@@ -0,0 +1,121 @@
+
+Sure is cloudy tonight, isn't it?
+
+[![](https://assets.ppy.sh/contests/186/banners/oniichan49.jpg)](https://assets.ppy.sh/contests/186/winners/oniichan49.png)
+
+**1210** voters hopped along to [**oniichan49**](https://osu.ppy.sh/users/20871620)'s jiangshi pippi and Aiko grooving out before a floating pumpkin audience. A witchy don is conducting this danse macabre, and we're all here for it... in spirit.
+
+[![](https://assets.ppy.sh/contests/186/banners/Triantafyllia.jpg)](https://assets.ppy.sh/contests/186/winners/Triantafyllia.png)
+
+Decoration is the name of the game in [**Triantafyllia**](https://osu.ppy.sh/users/17084594)'s piece, impressing **1122** voters. So much going on here between pippi and Mocha's incredibly ornate costumes and vibrant artistry really gunning for a beautiful collision of candy-like light and colour.
+
+[![](https://assets.ppy.sh/contests/186/banners/AyoLie.jpg)](https://assets.ppy.sh/contests/186/winners/AyoLie.png)
+
+UHHHHHH BRANES. [**AyoLie**](https://osu.ppy.sh/users/29186645) DREW ZOMBIE PIPPI. ZOMBIE MOCHA TOO. UUUURRRRRR. **1115** VOTERS LIKE VERY MUCH. BRAAAAANES. Also, a cute Chirou's chilling too. We're not entirely sure if she's a disembodied head or not, but we're going to pretend she is, because you know, Halloween. Wonderful piece!
+
+[![](https://assets.ppy.sh/contests/186/banners/Haru34_old.jpg)](https://assets.ppy.sh/contests/186/winners/Haru34_old.png)
+
+**1079** voter ghosts are waiting patiently in line to be hugged by [**Haru34_old**](https://osu.ppy.sh/users/8806199)'s pippi-mummy amidst a spooky violet-tinged moonscape. Why are these spirits so unbearably cute? It is a mystery.
+
+## peppy's Picks
+
+Long ago before time was counted, the Dean "peppy" Herbert once uttered deep within the hallowed crypts (of our internal Discord server) that there were too many good entries and not enough votes to go around. For thousands of years (approximately eleven seconds, we counted) did he contemplate, before deciding that he would pick a selection of his favorite entries to join our finalists on the main menu anyway. We've since discovered that if you gave him triple the number of ordinary votes, he'd still run out of them somehow.
+
+Thus, the Picks were born. And now, we all benefit.
+
+[![](https://assets.ppy.sh/contests/186/banners/nhyogu.jpg)](https://assets.ppy.sh/contests/186/winners/nhyogu.png)
+
+Normal Chirou. Orange Chirou. Evil Chirou. [**nhyogu**](https://osu.ppy.sh/users/7430439) Chirou. Orange Chirou. Evil Chirou. Orange Chirou. **EVIL CHIROU.**
+
+[![](https://assets.ppy.sh/contests/186/banners/Christi_Anna.jpg)](https://assets.ppy.sh/contests/186/winners/Christi_Anna.png)
+
+[**Christi_Anna**](https://osu.ppy.sh/users/11016739) gathered forth an assembly of mascots most stylish, including pippi, Chirou, Yuzu, the warring twins (Mani and Mari) plus Mocha and a don to enjoy some candelit Halloween festivities. Look at the cat!
```

Comment:
almost thought this was just another word i hadn't heard about but nope

![image](https://github.com/ppy/osu-wiki/assets/36758269/fea716bf-b7ab-446f-a078-dbe7c55da929)

```suggestion
[**Christi_Anna**](https://osu.ppy.sh/users/11016739) gathered forth an assembly of mascots most stylish, including pippi, Chirou, Yuzu, the warring twins (Mani and Mari) plus Mocha and a don to enjoy some candlelit Halloween festivities. Look at the cat!
```